### DISCUSSION TITLE
alternative to "pkexec /usr/bin/gdb"

### DISCUSSION THREAD

**Discussion by SiyuChern** [microsoft/vscode-discussions#863] [Extension Development QnA]
The issue has been reported in https://github.com/microsoft/WSL/issues/10626

I am following https://nadiah.org/2020/03/01/example-debug-mixed-python-c-in-visual-studio-code/ to set up the debugger in VScode.  However, pkexec /usr/bin/gdb seem not to work in WSL: 
Error getting authority: Error initializing authority: Could not connect: No such file or directory

Does anyone have an idea on this issue?

**Comment by isidorn** [comment]
I see there is good discussion in https://github.com/microsoft/WSL/issues/10626
So I am closing this in favour of that issue
